### ISSUE TITLE
HEC - Failed to connect to localhost port 8088: Connection refused

### ISSUE THREAD

**Issue by Vincent-GKC** [tmartin14/modern-app-dev-workshop#1]
Hi tmartin,

After adding the Boomerang.js on the index.html. My Buttercup-Go still not able send HEC (rum) traffic to Splunk. 

I tried:

`curl -k http://localhost:8088/services/collector -H 'Authorization: Splunk 00000000-0000-0000-0000-000000000003' -d '{"sourcetype": "testsourcetype", "event":"Hello, World!"}'`

But got error saying:
`0curl: (7) Failed to connect to localhost port 8088: Connection refused`

Then I tried using the same curl at my local machine. The test message can come thru OKAY.

Any thought?

Cheers,
Vincent

**Comment by tmartin14** [comment]
I'm not sure I understand the problem.   It sounds like you're running Splunk on localhost, but trying to run the curl from another machine but still posting to localhost.   If so, that won't work, you'll need to post to the machine where Splunk is running on port 8088.  

If you're still experiencing problems please let me know at which step of the lab you're having issues and I'll take a look.

**Comment by Vincent-GKC** [comment]
Thanks. I found out the issue is actually I am running the environment in a Ubuntu VM of my Windows OS. So when the Buttercup-go trying to forward HEC data to localhost:8088, it won't go, because localhost:8088 is actually my Windows OS, not the Splunk instance. 

I fixed the issue by adding a port forwarding option in my Windows machine. So localhost:8088 can actually goes to splunk.

Many thanks for you replied. And the lad is awesome!!


Cheers,
Vincent
